src/compiler/interface.jl

```diff
@@ -135,7 +135,7 @@ end
 
 Base.show(io::IO, ps::Grads) = print(io, "Grads(...)")
 
-@forward Grads.grads Base.getindex, Base.haskey
+@forward Grads.grads Base.getindex, Base.haskey, Base.iterate, Base.keys
```

Comment:
Think this needs a rebase because these forwards will conflict with the current master.